docs/LabVIEW_Debugging_Techniques.md

```diff
@@ -0,0 +1,128 @@
+### Debugging and Benchmarking
+
+Debugging and benchmarking is a normal process of code development. There are a variety of ways to debug and benchmark custom devices.
+
+#### LabVIEW Debugging Techniques
+
+Custom devices are written in LabVIEW code. Therefore it’s possible to develop, test and debug in LabVIEW before running the Custom Device Template Tool. In other words, you can use **[LabVIEW’s built-in debugging techniques](https://www.ni.com/en-us/support/documentation/supplemental/12/debugging-techniques-in-labview.html)** during development; and merge the LabVIEW code into the custom device framework after it matures.
+
+Since the custom device is one of many parts of the system definition, the behavior of the LabVIEW code within the custom device framework will likely differ from the stand-alone LabVIEW application, especially in regards to timing. As a result, you should benchmark the custom device inside of the VeriStand Engine.
+Once added to the system definition, custom devices have been fully integrated into VeriStand’s context. As a result, LabVIEW’s built-in debugging techniques are no longer available. Several techniques are available for debugging and benchmarking the custom device.
+ 
+#### Console Viewer
+
+A subcomponent of VeriStand RT Engine is the RT Console Viewer.
+You can install it to the execution host using Measurement and Automation Explorer. When installed, the component runs a small UDP daemon allowing the operator to view the console from a utility called the RT Console Viewer. You can access the RT Console viewer from **VeriStand » Workspace » Tools » Console Viewer**.	
+
+![](images/ConsoleViewer1.jpg)
```

Comment:
Please replace ConsoleViewer1.jpg with this updated picture:
![image](https://user-images.githubusercontent.com/62251693/132238422-523ba58b-6227-47ce-ac9a-7ed62795ef1f.png)